### PR TITLE
Fix the pos_getPendingUnlocks endpoint

### LINKED ISSUE #7974: Error when invoking pos_getPendingUnlocks
### Expected behavior
`pos_getPendingUnlocks` when invoked must return response.

### Actual behavior
Invoking the endpoint results in error.

### Steps to reproduce
```
./bin/run endpoint:invoke pos_getPendingUnlocks '{"address": "lsk37av46nmhr979948rvp3zx7n39xpetoqnqrxny"}' 
{"error":{"message":"Cannot read properties of undefined (reading 'aggregateCommit')"}}
```

### Which version(s) does this affect? (Environment, OS, etc...)
Lisk SDK `v6.0.0-alpha.10`

### PR DESCRIPTION
**Pull request by mosmartin**
### What was the problem?

This PR resolves #7974

### How was it solved?

- Updated the invoke() method in both in-memory channel and bus classes

### How was it tested?

- Updated unit tests

### CODE REVIEW

File: framework/src/controller/bus.ts [hunk 1]
```diff
@@ -97,6 +98,7 @@ export class Bus {
 		this._events = {};
 		this._channels = {};
 		this._rpcClients = {};
+		this._context = {};
```
Comment (shuse2) [inline framework/src/controller/bus.ts:101]: I think context changes on every call, so it's better not to keep the context as member. Also, this is never updated, so I think not needed?

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: framework/src/controller/bus.ts (hunk 1 above)
Comment (mosmartin) [reply framework/src/controller/bus.ts:101]: Yes, it's really not needed. Need to pass context in other functions that call the invoke() method.

**Review by shuse2 — COMMENTED** [review]
LGTM, but i think the path where context is not set is not tested

**Comment by mosmartin** [comment]
>

**Comment by shuse2** [comment]
@mosmartin I think L217 is called with `context`?
Can you check the reason of decrease for `codecov/patch` and you can see the uncovered part https://app.codecov.io/gh/LiskHQ/lisk-sdk/pull/7988

**Comment by mosmartin** [comment]
> @mosmartin I think L217 is called with `context`? Can you check the reason of decrease for `codecov/patch` and you can see the uncovered part https://app.codecov.io/gh/LiskHQ/lisk-sdk/pull/7988

@shuse2 , the link displays a blank page. Are you able to see the content?

**Review by shuse2 — APPROVED** [review]

**Review by has5aan — APPROVED** [review]

**Pull request merged**